automatic/flashplayeractivex/update.ps1

```diff
@@ -1,7 +1,9 @@
-
+﻿
 import-module au
+. "$PSScriptRoot\..\..\scripts\Get-Padded-Version.ps1"
 
 $releases = "https://get.adobe.com/en/flashplayer/" # URL to for GetLatest
+$padVersionUnder = '24.0.1'
```

Comment:
This would be needed to be changed manually (by whoever fixes the package) when/if the last released package is equal to or above that version.